ui/restmodel/src/main/scala/pl/touk/nussknacker/restmodel/displayedgraph/ProcessStatus.scala

```diff
@@ -78,36 +78,42 @@ object ProcessStatus {
   }
 
   def simpleErrorShouldRunning(deployedVersionId: Long, user: String, previousState: Option[ProcessState]): ProcessStatus = simpleError(
+    errorStatus = SimpleStateStatus.RunningError,
```

Comment:
Here process is not running

Reply:
ProcessState is not running but ProcessAction is deployed, so.. We should treat as running state.

 I've tested it by:
- deploy process 
- cancel process in flink

Then message should be 'Process should be running' and we should can do Cancle and Deploy action.

Reply:
After consider I've added DeployedWithError status.